File: web-services/services/auditing/src/main/java/com/cryptocaddy/services/auditing/model/request/RequestUserData.java
package com.cryptocaddy.services.auditing.model.request;

import java.util.List;

public class RequestUserData {

    public boolean requestExchangeWallets;
    public List<String> limitExchangeWallets;

    public boolean requestExchangeTrades;
    public List<String> limitExchangeTrades;

    public RequestUserData() {

    }

}

File: web-services/services/auditing/src/main/java/com/cryptocaddy/services/auditing/dto/request/RequestUserData.kt
package com.cryptocaddy.services.auditing.dto.request

class RequestUserData {

    var requestExchangeWallets: Boolean = false
    var limitExchangeWallets: List<String>? = null

    var requestExchangeTrades: Boolean = false
    var limitExchangeTrades: List<String>? = null

}
